docs/en/operation/Remote-Logging.md

```diff
@@ -67,7 +67,7 @@ The following lines would need to be added to `conf/alluxio-env.sh` to enable re
 
 ```bash
 ALLUXIO_LOGSERVER_HOSTNAME=AlluxioLogServer
-ALLUXIO_LOGSERVER_PORT=45600
+ALLUXIO_LOGSERVER_PORT="45600"
```

Comment:
will it cause the regression to other deployments?

Reply:
why is this needed?

Reply:
I thought I put an explanation on this PR this line but it seemed to be a github webpage issue that my comment didn't go through...

The reason is in K8s configmap, this value has to be string instead of number, otherwise a parse error will be thrown. I'm changing the  non-k8s instruction to use a string value too so both places are consistent.

You can find the k8s part instruction using a string value [here](https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Kubernetes.html#enable-remote-logging) in the `kubectl` section. I tested manually on physical machine too so no regression is expected.

Reply:
@apc999 @HelloHorizon